docs/recipes-common-issues-and-their-solutions.rst

```diff
@@ -127,56 +127,6 @@ It features maintaining any metadata, table of contents and links contained in t
 
 
 
-How to Deal with Messages Issued by :title:`MuPDF`
```

Comment:
Shouldn't we re-write this section rather than delete it?

Reply:
It's been rewritten and moved to section `MuPDF errors and warnings` in apps3.rst.